packages/javascript/bh-shared-ui/src/components/AssetGroupManagement/AssetGroupSelectorObjectSelect.tsx

```diff
@@ -76,38 +69,44 @@ const AssetGroupSelectorObjectSelect: FC<{ seeds: SelectorSeedRequest[] }> = ({
         refetchOnWindowFocus: false,
     });
 
-    const seedsQuery = useQuery({
-        queryKey: ['tier-management', 'tags', tagId, 'selectors', selectorId, 'seeds'],
-        queryFn: async ({ signal }) => {
+    useEffect(() => {
+        if (!previewQuery.data) return;
+
+        setResults(previewQuery.data || []);
+    }, [previewQuery.data, setResults]);
+
+    useEffect(() => {
+        const cypherQuery = async () => {
```

Comment:
Let's add an early return if there are no seeds so as to avoid making a request
```suggestion
        if (seeds.length === 0) return;
        const cypherQuery = async () => {
```